modules/platform/nuxeo-apidoc-server/nuxeo-apidoc-core/src/main/java/org/nuxeo/apidoc/api/BundleInfo.java

```diff
@@ -91,4 +96,13 @@
      */
     Blob getParentReadme();
 
+    /**
+     * Returns the deployment order as indicated by the runtime context.
+     *
+     * @since 11.1
+     */
+    Long getDeploymentOrder();
+
+    void setDeploymentOrder(Long order);
```

Comment:
Setter could use a `@since 11.1` too for consistency.

Reply:
indeed!